rllib/evaluation/rollout_worker.py

```diff
@@ -1150,7 +1165,8 @@ def remove_policy(
             *,
             policy_id: PolicyID = DEFAULT_POLICY_ID,
             policy_mapping_fn: Optional[Callable[[AgentID], PolicyID]] = None,
-            policies_to_train: Optional[List[PolicyID]] = None,
+            policies_to_train: Optional[Union[Container[PolicyID], Callable[
```

Comment:
I don't know if it's worth it to define some names for the 2 func types here, kinda long, and used multiple times.
totally up to you.